targets/TARGET_STM/TARGET_STM32F0/TARGET_NUCLEO_F072RB/device/TOOLCHAIN_GCC_ARM/NUCLEO_F072RB_retarget.c

```diff
@@ -0,0 +1,55 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018, STMicroelectronics
```

Comment:
Could you rename the file name to be more generic ? and then copy it to NUCLEO_F070 also ?

Reply:
Done